alterclientquotas.go

```diff
@@ -0,0 +1,134 @@
+package kafka
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/segmentio/kafka-go/protocol/alterclientquotas"
+)
+
+// AlterClientQuotasRequest represents a request sent to a kafka broker to
+// alter client quotas.
+type AlterClientQuotasRequest struct {
+	// Address of the kafka broker to send the request to.
+	Addr net.Addr
+
+	// List of client quotas entries to alter.
+	Entries []AlterClientQuotaEntry
+
+	// Whether the alteration should be validated, but not performed.
+	ValidateOnly bool
+}
+
+type AlterClientQuotaEntry struct {
+	// The quota entities to alter.
+	Entities []AlterClientQuotaEntity
+
+	// An individual quota configuration entry to alter.
+	Ops []AlterClientQuotaOps
+}
+
+type AlterClientQuotaEntity struct {
+	// The quota entity type.
+	EntityType string
+
+	// The name of the quota entity, or null if the default.
+	EntityName string
+}
+
+type AlterClientQuotaOps struct {
+	// The quota configuration key.
+	Key string
+
+	// The quota configuration value to set, otherwise ignored if the value is to be removed.
+	Value float64
+
+	// Whether the quota configuration value should be removed, otherwise set.
+	Remove bool
+}
+
+type AlterClientQuotaResponseQuotas struct {
+	// The error code, or `0` if the quota alteration succeeded.
+	ErrorCode int16
```

Comment:
In the client responses we usually wrap these details on an `error` typed field which can be constructed with https://github.com/segmentio/kafka-go/blob/5e7b724b8a23e2dc5adfb508c00dec5307719692/error.go#L658